en/setup/_addition_dev_tools.txt

```diff
@@ -10,6 +10,8 @@ Download and install the [QGroundControl Daily Build](https://docs.qgroundcontro
 
 The development team often use:
 
+* [Visual Studio Code](https://code.visualstudio.com/): quite new, popular open source IDE
+* [eclipse for C/C++](https://www.eclipse.org/downloads/eclipse-packages/): very feature rich Java based IDE
```

Comment:
eclipse -> Eclipse

These are fine if these are genuinely used by the team

Reply:
I wanted to write it like in their logo:
![eclipse](https://www.eclipse.org/downloads/packages/sites/all/themes/solstice/_themes/solstice_packages/logo.png)

Reply:
fixed